src/components/Comment.jsx

```diff
@@ -1,7 +1,7 @@
 const Comment = ({comment}) => {
 return (
     <article className="comment">
-        <h4>{comment.author}</h4>
+        <h3>{comment.author}</h3>
         <p>at {comment.created_at}</p>
```

Comment:
I would think about how we would want to display our created at as it is not very readable at the moment